.github/workflows/release-binaries.yml

```diff
@@ -19,15 +19,26 @@ jobs:
         run: |
           set -e
           set -o pipefail
-          cat bin/darwin-x64 | gzip --best >bin/darwin-x64.gz
-          cat bin/darwin-arm64 | gzip --best >bin/darwin-arm64.gz
-          # cat bin/freebsd-x64 | gzip --best >bin/freebsd-x64.gz
-          cat bin/linux-arm | gzip --best >bin/linux-arm.gz
-          cat bin/linux-arm64 | gzip --best >bin/linux-arm64.gz
-          cat bin/linux-ia32 | gzip --best >bin/linux-ia32.gz
-          cat bin/linux-x64 | gzip --best >bin/linux-x64.gz
-          cat bin/win32-ia32 | gzip --best >bin/win32-ia32.gz
-          cat bin/win32-x64 | gzip --best >bin/win32-x64.gz
+          # ffmpeg
+          cat bin/ffmpeg-darwin-x64 | gzip --best >bin/ffmpeg-darwin-x64.gz
```

Comment:
While we're at it, I think this can be simplified as `for file in bin/{ffmpeg,ffprobe}-{darwin,linux,win32,freebsd}-{arm64,arm,x64,ia32}; do gzip -k --best "$file"; done`.

Reply:
Good suggestion, I updated it a bit since from what I was able to test it would fail on files it couldn't fine with the `-e` option